config/root-config.in

```diff
@@ -11,43 +11,43 @@ if [ `uname` = "AIX" ]; then
 fi
 
 # work around readlink versions not having -f option
-fullpath1=`$readlink "$0"`
+fullpath1=`$readlink $0`
```

Comment:
All of these changes (fullpath / fullpath1) should probably be kept? Just the ones below (bindir etc) as causing problems?

Reply:
OK, let's try that...